docs(content): Add Doc for Persistence Connection Logging (#1530)
camunda/camunda-docs-manual: content/update/minor/719-to-720/_index.md

12. For developers: [Camunda Platform Run requires JDK 17](#camunda-platform-run-requires-jdk-17)
13. For developers: [Update Alpine Base Docker Image from version 3.15 to 3.18](#update-alpine-base-of-camunda-docker-images-from-version-3-15-to-3-18)
14. For developers: [Discontinue support for Velocity, XSLT, and XQuery template engines](#discontinue-support-for-velocity-xslt-and-xquery-template-engines)

> 15. For developers: [Changes on Persistence Connection Exception Logging](#changes-on-persistence-connection-exception-logging)

Reviewer: 🔧 We use sentence case spelling for headers and titles as described in our [writing style guide](https://confluence.camunda.com/display/HAN/Writing+Style+Cheat+Sheet#WritingStyleCheatSheet-Formatting,organizationandstructureforconceptualpiecesandimplementationsteps).
```suggestion
15. For developers: [Changes on persistence connection exception logging](#changes-on-persistence-connection-exception-logging)
```

```

We are looking for maintainers for the template engine extensions. Feel free to reach out to us [via the forum](https://forum.camunda.io/c/camunda-platform-7-topics/39) if you are interested.

> # Changes on Persistence Connection Exception Logging

Reviewer: 🔧 We use sentence case spelling for headers and titles as described in our [writing style guide](https://confluence.camunda.com/display/HAN/Writing+Style+Cheat+Sheet#WritingStyleCheatSheet-Formatting,organizationandstructureforconceptualpiecesandimplementationsteps).
```suggestion
# Changes on persistence connection exception logging
```

> All persistence connection exceptions (having a cause of **SQL class 8**) will be logged with ERROR logging level from now on.

Reviewer: 🔧 The linked article details what SQL states there are. I would argue it's enough to mention `connection exception` here since this is also what's used on that page 🙂 
```suggestion
All persistence connection exceptions will be logged with logging level ERROR instead of WARN from now on.
```

At the same time, we acknowledge there might be use cases where our users might favour to keep the legacy behaviour despite our recommendation. If that is the case, you can always turn off the feature by setting
the feature flag `enforceHistoryTimeToLive` to `false`. For more information, checkout the new parameter description under <a href="{{< ref "/reference/deployment-descriptors/tags/process-engine#configuration-properties" >}}">Configuration Properties</a>.

# Changes on persistence connection exception logging

All persistence connection exceptions will be logged with logging level ERROR instead of WARN from now on.

> You can read more about SQL connection exceptions & SQL classes [here](https://en.wikipedia.org/wiki/SQLSTATE).

Reviewer 2: Should we mention this is only relevant for API calls made via the REST API? This paragraph sounds like it affects the Java API as well.

Author: That's a fair point. Here is a [change](https://github.com/camunda/camunda-docs-manual/pull/1537) which states explicitly the REST API on the title.
